yourgov/components/FormMobileFoodVendorPermit/task/FormStep7.tsx

```diff
@@ -1,72 +1,210 @@
-					control={control}
-					name="cuisine"
-					render={({ field: { ref, ...field } }) => (
-						<ComboboxMulti
-							label="TODO"
-							hint="TODO"
-							inputRef={ref}
-							options={cuisineOptions}
-							required
-							{...field}
-							id="cuisine"
-							invalid={Boolean(typeCorrectedErrors.cuisine?.message)}
-							message={typeCorrectedErrors.cuisine?.message}
-							maxWidth="xl"
-							block={false}
-						/>
-					)}
-				/>
-				<StepActions />
+			<Stack gap={2}>
+				{showErrorMessage && (
+					<PageAlert focusOnMount title="No employees added" tone="error">
+						<Text as="p">
+							You need to add some staff. You’re amazing, but you can’t do this
+							alone. Life is better with friends and team mates.
+						</Text>
+					</PageAlert>
+				)}
+				<H2 id="list-of-employees">List of employees</H2>
+				{showAddedEmployeeMessage && (
+					<SectionAlert
+						focusOnMount
+						onClose={() => setShowAddedEmployeeMessage(false)}
+						title={`${addedEmployee?.firstName} ${addedEmployee?.lastName} has been added as an
+							employee`}
+						tone="success"
+					/>
+				)}
+				{showRemovedEmployeeMessage && (
+					<SectionAlert
+						focusOnMount
+						onClose={() => setShowRemovedEmployeeMessage(false)}
+						title={`${removedEmployee} has been removed as an
+							employee`}
+						tone="success"
+					/>
+				)}
+				{step7State?.employee && step7State.employee.length > 0 ? (
+					<TableWrapper>
+						<Table aria-labelledby="list-of-employees">
+							<TableHead>
+								<TableRow>
+									<TableHeader>Employee name</TableHeader>
+									<TableHeader>Email address</TableHeader>
+									<TableHeader>Action</TableHeader>
+								</TableRow>
+							</TableHead>
+							<TableBody>
+								{step7State.employee.map(
+									(employee) =>
+										employee?.id && (
+											<TableRow key={employee.id}>
+												<TableCell id={employee.id}>
+													{employee.firstName} {employee.lastName}
+												</TableCell>
+												<TableCell>{employee.email}</TableCell>
+												<TableCell>
+													<Button
+														aria-describedby={employee.id}
+														onClick={() => {
+															setEmployeeToRemove(employee);
+														}}
+														variant="text"
+													>
+														Remove
+													</Button>
+												</TableCell>
+											</TableRow>
+										)
+								)}
+							</TableBody>
+						</Table>
+					</TableWrapper>
+				) : (
+					<Stack gap={1}>
+						<AvatarIcon color="muted" size="lg" />
+						<H3>No employees added</H3>
+						<Text as="p">You must add an employee to proceed.</Text>
+					</Stack>
+				)}
+				<ButtonLink
+					alignSelf="start"
+					href={taskFormSteps[6].items && taskFormSteps[6].items[0].href}
+					iconBefore={PlusIcon}
+					size="sm"
+					variant="secondary"
+				>
+					Add employee
+				</ButtonLink>
 			</Stack>
+			<form onSubmit={onSubmit} noValidate>
+				<StepActions />
```

Comment:
It feels like we should dynamically update the text in the modal that renders when you press "Cancel" on this step. You won't actually lose an employee that you've added even if you haven't pressed "Save and continue". Or maybe we clear the table? Or even just those items added since last save?

![image](https://github.com/user-attachments/assets/6e2546de-8a82-47e1-bb35-c56725a47787)

Reply:
Good pickup, but only because we don't have the "staging" area for the employees added. I think as a future task we should do that instead of customising the modal.

Reply:
Ok, that sounds good. So long as it's clear to users 🙌